aws/resource_aws_servicecatalog_portfolio_principal_association.go

```diff
@@ -0,0 +1,165 @@
+package aws
+
+import (
+	"fmt"
```

Comment:
small nit, can you split the deps to go packages and external dep packages like other resources?

Reply:
done - all files (check-in to come)